uvicorn/workers.py

```diff
@@ -75,6 +77,11 @@ def run(self) -> None:
         server = Server(config=self.config)
         loop = asyncio.get_event_loop()
         loop.run_until_complete(server.serve(sockets=self.sockets))
+        # Exit with status 3 when worker starts failed, so Gunicorn
+        # can shut it down to avoid infinite start/stop cycles.
+        # See: https://github.com/encode/uvicorn/issues/1066
```

Comment:
@euri10 Do you want us to keep this comment? I feel like it's unnecessary after the modifications, but I'll let you decide :kissing:

Reply:
yes let's keep it please: @fanchunke1991 found the solution after some serious digging inside gunicorn's code and if one can still use `git-blame` or that vim god-like key `nmap <silent><leader>g :call setbufvar(winbufnr(popup_atcursor(split(system("git log -n 1 -L " . line(".") . ",+1:" . expand("%:p")), "\n"), { "padding": [1,1,1,1], "pos": "botleft", "wrap": 0 })), "&filetype", "git")<CR>` to see where that comes from I think that down the road it will be easier for future readers to track the reason !